solr/core/src/java/org/apache/solr/packagemanager/PackageManager.java

```diff
@@ -173,7 +173,7 @@ public void uninstall(String packageName, String version)
         String.format(Locale.ROOT, "/package/%s/%s/%s", packageName, version, "manifest.json"));
     for (String filePath : filesToDelete) {
       DistribFileStore.deleteZKFileEntry(zkClient, filePath);
-      String path = "/api/cluster/files" + filePath;
+      String path = "/api/cluster/filestore/files" + filePath;
```

Comment:
i like the more verbose path I think...  Can a filestore have anything under than files?   If not, what about just `"/api/cluster/filestore"`

Reply:
> Can a filestore have anything under than files?

It can: "metadata" and "commands" are both under the "filestore" path as siblings to "files".